ci/scripts/e2e-source-test.sh

```diff
@@ -59,6 +59,23 @@ export PGHOST=db PGPORT=5432 PGUSER=postgres PGPASSWORD=postgres PGDATABASE=cdc_
 createdb
 psql < ./e2e_test/source/cdc/postgres_cdc.sql
 
+# to test upper case database
+psql -c "CREATE DATABASE \"UpperDb\";"
+
+export PGHOST=db PGPORT=5432 PGUSER=postgres PGPASSWORD=postgres PGDATABASE=UpperDb
+psql -d UpperDb -c "
+CREATE SCHEMA \"UpperSchema\";
+CREATE TABLE \"UpperSchema\".\"Orders\" (
+    id int PRIMARY KEY,
+    name varchar
+);
+INSERT INTO \"UpperSchema\".\"Orders\" VALUES (1, 'happy');
```

Comment:
You can refer to the test in `e2e_test/source/cdc_inline` to write the creation in slt.